cmd/openshift-install/create.go

```diff
@@ -427,8 +427,8 @@ func logComplete(directory, consoleURL string) error {
 		return err
 	}
 	logrus.Info("Install complete!")
-	logrus.Infof("Run 'export KUBECONFIG=%s' to manage the cluster with 'oc', the OpenShift CLI.", kubeconfig)
-	logrus.Infof("The cluster is ready when 'oc login -u kubeadmin -p %s' succeeds (wait a few minutes).", pw)
+	logrus.Infof("To access the cluster as the system:admin user, run 'export KUBECONFIG=%s'", kubeconfig)
```

Comment:
I think you want to mention `oc` in this line.  Maybe:

```go
logrus.Infof("To access the cluster with 'oc' as the system:admin user, run 'export KUBECONFIG=%s'", kubeconfig)
```

Reply:
looks good! thanks, updated